docs/developer/decision-records/2022-10-31-aggregate-service-layer/README.md

```diff
@@ -14,7 +14,7 @@ For `AggregateService` we intend a component that provides a clear and simple in
 `Asset`s, `ContractNegotiation`s and so on) encapsulating some logic and features, like business validation, 
 transactional context, exception handling, ...
 
-Currently they are located into the `data-management-api`. Moving them to a separate layer will permit other kinds of
+Currently they are located into the `management-api`. Moving them to a separate layer will permit other kinds of
```

Comment:
IMO, decision records should be excluded from refactorings as they represent a snapshot of a specific date in time.

Reply:
ops, that was an overshight